### ISSUE TITLE
Create space dialog has extra space on the bottom

### ISSUE THREAD

**Issue by jkandasa** [openshiftio/openshift.io#1817]
When creating new space, There is a popup to enter `Space` name. I see extra space at the bottom of the dialog.
![image](https://user-images.githubusercontent.com/1004403/34771994-85ac8140-f62c-11e7-90c8-1fa9da44067f.png)

**Comment by joshuawilson** [comment]
@catrobson do we have plans to update the design for the space creation part of the wizard?

**Comment by catrobson** [comment]
@joshuawilson yes, but we've only done a bit of design thinking about this so there are not existing designs to point to right now.  We hope to address this soon as a related step of the wizard design process.

**Comment by qodfathr** [comment]
See also https://github.com/openshiftio/openshift.io/issues/1542#issuecomment-360143574 which suggests this blank area could be used for validation error messages.

**Comment by andrewazores** [comment]
Is this still valid? I think this wizard is being (has been?) phased out.
